docs-guides/source/convert-llm.md

```diff
@@ -0,0 +1,153 @@
+# Converting a Large Language Model
```

Comment:
You dont need to be generic here. Just say converting openELM model. 
This page is about a specific model only , so the title should reflect that.

Reply:
Re-titled as suggested
```
Converting an Open Efficient Language Model
```